.github/workflows/publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 3.1.100-preview1-014459
+        dotnet-version: 3.1.0-preview3.19555.2

```

Comment:
It should be `3.1.100-preview3-014645`. What you added here is the package Version while this entry requires the SDK version. Please update it.

Reply:
Thanks